Pipfile

```diff
@@ -9,16 +9,24 @@ requests-mock = "*"
 coveralls = "*"
 coverage = "*"
 pytest-cov = "*"
+flake8 = "*"
+pylama = "*"
+black = "*"
+bandit = "*"
 
 [packages]
 archivessnake = "*"
 click = "*"
 jsonpatch = "*"
 jsonpointer = "*"
 structlog = "*"
+isort = "*"
 
 [requires]
 python_version = "3.9"
 
 [scripts]
 asaps = "python -c \"from asaps.cli import main; main()\""
+
+[pipenv]
+allow_prereleases = true
```

Comment:
This was required to install `black` since it's still prerelease. I was getting errors from `pipenv` and found these: https://github.com/microsoft/vscode-python/issues/5171 and https://github.com/psf/black/issues/517

Reply:
Ah, cool. Would you mind adding that as a comment in the Pipfile so we know why, and also as a reminder to fix it when black is ready?

Reply:
Will do!

Reply:
Actually I'm rethinking this. Instead of allowing pre-releases for all packages (generally not a great practice), could we just pin Black to the current version? I don't love that either, but I like it better than possibly installing pre-releases for other packages that aren't ready for primetime.

Reply:
That worked